datasetinsights/io/config_handler.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import re
+
+from datasetinsights.io.exceptions import InvalidOverrideKey
+
+logger = logging.getLogger(__name__)
+_REGEX_PTRN = r"^([:/\w_+-.%]+=[:/\w_+-.%]+)(\s[:/\w_+-.%]+=[:/\w_+-.%]+)*$"
+_EQUAL_STR = "="
+
+
+def override_config(override=None, config=None):
+    """override config params of config YAML, if override
+    key doesn't exist it will throw Non-existent key error .
+
+    Args:
+        override: string of key-value pairs
+        config: config object of type yacs.config.CfgNode
+    Examples:
+         override="optimizer.args.lr=0.00005 pretrained=False"
+    """
+
+    pattern = re.compile(_REGEX_PTRN)
```

Comment:
Can the validation of override string separated from `override_config` method as it's own function?